app/language/Dates.scala

```diff
@@ -25,4 +25,22 @@ object Dates {
 
   def formatDate(date: LocalDate): String = `format d MMMM y`.format(date)
 
+  def getMonthlyDateFormatted(localDate: LocalDate): String = {
```

Comment:
Probably this should be parametried by language and return Welsh variant as well